contracts/prize-strategy/PrizeSplit.sol

```diff
@@ -83,26 +83,26 @@ abstract contract PrizeSplit is IPrizeSplit, Ownable {
     }
 
     /// @inheritdoc IPrizeSplit
-    function setPrizeSplit(PrizeSplitConfig memory prizeStrategySplit, uint8 prizeSplitIndex)
+    function setPrizeSplit(PrizeSplitConfig memory _prizeStrategySplit, uint8 _prizeSplitIndex)
         external
         override
         onlyOwner
     {
-        require(prizeSplitIndex < _prizeSplits.length, "PrizeSplit/nonexistent-prizesplit");
-        require(prizeStrategySplit.target != address(0), "PrizeSplit/invalid-prizesplit-target");
+        require(_prizeSplitIndex < _prizeSplits.length, "PrizeSplit/nonexistent-prizesplit");
+        require(_prizeStrategySplit.target != address(0), "PrizeSplit/invalid-prizesplit-target");
```

Comment:
Should be named `_prizeSplitStrategy` if we want to follow the convention introduced by `_prizeSplitIndex`.